### ISSUE TITLE
Record and stream

### ISSUE THREAD

**Issue by Volont84** [jitsi/docker-jitsi-meet#13]
Hi!
It is a great solution easy to use and work just fine.
But i have an issue with recording. As far i understand it is not possible to record or stream your conference. I tried to create jibri docker image but no luck for now. Maybe there is some another way to record?

**Comment by dimm0** [comment]
Do I understand it right that we can send the video and audio streams from jitsi to ffmpeg, which will generate a resulting file/stream for us?

https://unix.stackexchange.com/questions/233832/merge-two-video-clips-into-one-placing-them-next-to-each-other

**Comment by aaronkvanmeerten** [comment]
That's incorrect.  The jitsi videobridge does not generate video streams.  It simply forwards UDP packets.
